armi/settings/fwSettings/globalSettings.py

```diff
@@ -154,6 +156,28 @@ def defineSettings() -> List[setting.Setting]:
             "block. Used for axial mesh refinement.",
             schema=vol.All(vol.Coerce(int), vol.Range(min=0, min_included=False)),
         ),
+        setting.Setting(
+            CONF_PRIMARY_ASSEMBLY_CONSERVE,
+            default="fuel",
```

Comment:
I recommend adding a setter to this setting that transforms the inputs into valid flags so that we arent passing string inputs around.